app/controllers/application_controller.rb

```diff
@@ -22,8 +22,4 @@ def find_bookmark(id)
   def tutorial_name(id)
     Tutorial.find(id).title
   end
-
-  def four_oh_four
```

Comment:
What was this even for?

Reply:
It was a bad implementation of the normal `require_admin`, instead raising an exception and breaking the app.